doc-src/SASS_REFERENCE.md

```diff
@@ -884,6 +884,43 @@ is compiled to:
     p {
       font: 12px/30px; }
 
+##### Subtraction and `-`
+{#subtraction-and-hyphen}
+
+CSS allows `-` to appear in identifiers. SassScript must support this syntax,
+and also allow `-` to be used for subtraction. To ensure that a `-` is
+interpreted as subtraction, keep it spaced out from the operands. For example:
+
+    p {
+      margin: 20px-10px;         // Plain CSS, no subtraction
+      padding: 20px - 10px 40px; // hyphen is spaced out, does subtraction
+    }
+
+is compiled to:
+
+    p {
+      margin: 10px;
```

Comment:
This is incorrect. Sass will produce `20px-10px` here, because `px-10px` is a valid identifier and thus a valid name for a unit.